src/build/rustc.rs

```diff
@@ -70,7 +73,11 @@ pub fn rustc(
 
     let buf = Arc::new(Mutex::new(vec![]));
     let err_buf = buf.clone();
-    let args = args.to_owned();
+    let args: Vec<_> = if cfg!(feature = "clippy") {
+        args.iter().map(|s| s.to_owned()).chain(Some("-Aclippy".to_owned())).collect()
```

Comment:
Slight hackery. Clippy automatically turns certain lints to warn by default. This undoes that. You can reenable them in the code though.